### ISSUE TITLE
Workaround for regression in H2 generator where primary keys are no longer found

### ISSUE THREAD

**Issue by masc3d** [jOOQ/jOOQ#7330]
### Expected behavior and actual behavior:
should generate @Id annotations, but it doesn't

### Steps to reproduce the problem:
upgrade to h2-1.4.197, generate classes

### Versions:
3.10.5

### Example

running against h2-1.4.196 generates

```
	@Id
	@Column(name = "ID", unique = true, nullable = false, precision = 19)
	public Long getId() {
		return this.id;
	}
```

running against the same database using h2-1.4.197 generates

```
	@Column(name = "ID", nullable = false, precision = 19)
	public Long getId() {
		return this.id;
	}
```

**Comment by lukaseder** [comment]
Thank you very much for your report. Indeed, the query producing primary key information in jOOQ-meta no longer works. Will investigate immediately.

**Comment by lukaseder** [comment]
OK, this is due to a regression in H2: https://github.com/h2database/h2database/issues/1000

As I'm going to release 3.10.6 later today, I guess we can work around this problem in jOOQ

**Comment by lukaseder** [comment]
Fixed for jOOQ 3.11. Will merge to 3.10.6 (#7332) and 3.9.7 (#7333)
